common/build/build-common/.changeset/rich-candles-tan.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/build-common": minor
```

Comment:
@tylerbutler Should this be `major`, since it is technically a breaking change?

Reply:
It should be whatever you're releasing it as. Usually we direct major changes to next and minor changes to main, but independent packages are a little different. I would prefer to be accurate and treat this as major, but there's been some pushback (around the typescript target).

Reply:
Hmm, is there a specific reason for the pushback? It would be really easy to break existing builds by publishing a new minor version of this package. I think this needs to be a major bump.

Reply:
I could also target next if we specifically want to avoid needing to update dependencies for a major version to propagate the TypeScript version changes (though I very much question not treating that as a breaking change as well).